src/commands/roll.rs

```diff
@@ -126,6 +129,13 @@ pub async fn roll(
         ),
     );
 
+    let roll_str = rolls_str(&rolls, mastery);
+    let mut message = format!("# {roll_str}");
```

Comment:
Not sure if copying this here was intentional - `message` gets reinitialized on line 140, so it won't use the value you create and write to here. 
In fact, since we want this roll name to be printed on the top, this will have to move around a bit, so the message will need to start empty and then each part gets written to it. 
```rust
let mut message = String::new();
if let Some(roll_name) = name {
    ...
}
let rolls_str = ...;
write!(message, "# {roll_str}").unwrap();

```

Reply:
Yeah that was copied without intention